### DISCUSSION TITLE
github上线了新徽章！具体是怎么获得的呀

### DISCUSSION THREAD

**Discussion by xgdyp** [X-lab2017/open-wonderland#84] [6️⃣ 提问题] [answered]
![image](https://user-images.githubusercontent.com/37795442/173071906-ff5ce29a-925f-47a8-ac94-f826e9d80d1d.png)

**Comment by xgdyp** [comment, marked as answer]
github博客里有详细的介绍～
https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

**Reply by will-ww** [reply]
> github博客里有详细的介绍～ https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/

自问自答了，哈哈~

**Comment by xgdyp** [comment]
又[更新了](https://github.com/github-community/community/discussions/20443)
Starstruck: appear when you create a repository and get many stars.
Quickdraw: appear when you create a new PR and they are lost within 5 minutes of opening
Pair Extraordinaire: co-authoring commits on merged PR
